docs/source/index.rst

```diff
@@ -57,6 +67,17 @@ On Ubuntu 14.04 (Trusty):
   :linenos:
   :lines: 20-
 
+System Requirements
+===================
+
+Although Cartographer may run on other systems, it is confirmed to be working
```

Comment:
Would rather not speculate. It's possible that contributions will come in to support arbitrary architectures. Nothing fundamentally prevents us from working on 32-bit.